<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1050</width>
    <height>709</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="1">
     <widget class="QStackedWidget" name="stackedWidget">
      <widget class="QWidget" name="page_6"/>
      <widget class="QWidget" name="page">
       <widget class="QPushButton" name="pushButton">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>450</y>
          <width>450</width>
          <height>40</height>
         </rect>
        </property>
        <property name="text">
         <string>进入管理员模式</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>370</y>
          <width>451</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>13</pointsize>
         </font>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
        <property name="placeholderText">
         <string>请输入密码</string>
        </property>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>30</y>
          <width>720</width>
          <height>331</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <pointsize>17</pointsize>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>ArrowCursor</cursorShape>
        </property>
        <property name="acceptDrops">
         <bool>false</bool>
        </property>
        <property name="toolTip">
         <string/>
        </property>
        <property name="whatsThis">
         <string>使用说明</string>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>       您可以通过本页面进入管理员模式，在下面的文本框中输入正确的密码，即可进入管理员模式，对航班信息进行统一的查询管理，添加航班，修改航班信息，取消航班，以及对航班数据进行任意的合法操作，对此系统有任何的观点，欢迎您来提出意见和反馈，感谢您的使用</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_5">
       <widget class="QPushButton" name="pushButton_2">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>450</y>
          <width>450</width>
          <height>40</height>
         </rect>
        </property>
        <property name="text">
         <string>进入用户模式</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>40</y>
          <width>720</width>
          <height>291</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>17</pointsize>
         </font>
        </property>
        <property name="text">
         <string>你好，欢迎使用春秋航班航班信息查询系统通过本系统用户可进行航班信息的查询了解到，指定航班的详细信息查询方式可以通过使肥城市起飞，城市和抵达城市进行搜寻相关符合条件的航班，同时可进行购买操作，再次见面下，您不需要进行登录即可使用。n</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
        <property name="margin">
         <number>0</number>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_2">
       <widget class="QPushButton" name="pushButton_3">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>450</y>
          <width>450</width>
          <height>40</height>
         </rect>
        </property>
        <property name="text">
         <string>退出系统</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>30</y>
          <width>721</width>
          <height>341</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>17</pointsize>
         </font>
        </property>
        <property name="text">
         <string>您好，欢迎使用春秋航空航班信息查询管理系统，本系统基于C加加语言，qt界面设计进行开发，可在windows MAC等环境下进行运行，管理系统当中包含管理员模式和用户模式相关，后台工作人员可以根据相关的密码进入管理员模式，对航班信息进行添加修改，删除查询等操作，有超级管理员权限用户模式无需登录，可查询相关城市所含有的航班以及其详细信息，在使用过程当中，欢迎您来提出任何意见和反馈，感谢您的使用
</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QTreeWidget" name="treeWidget">
      <property name="maximumSize">
       <size>
        <width>180</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
        <kerning>true</kerning>
       </font>
      </property>
      <property name="cursor" stdset="0">
       <cursorShape>OpenHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <attribute name="headerVisible">
       <bool>false</bool>
      </attribute>
      <column>
       <property name="text">
        <string notr="true">1</string>
       </property>
      </column>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1050</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
